packages/orchestrator-ui-components/package.json

```diff
@@ -52,6 +52,7 @@
         "object-hash": "^3.0.0",
         "prism-themes": "^1.9.0",
         "react-diff-view": "^3.2.0",
+        "react-draggable": "^4.4.6",
```

Comment:
Why not using the EUI built-in way of resizing instead of adding a new dependency?
https://eui.elastic.co/#/layout/resizable-container#custom-resize-logic
That one is also more a11y-compliant

Reply:
That component doesn't have a standard callback that we need to use to update the column widths after we are done dragging. Also we have moved the table outside of EUI so it doesn't feel logical to move components back in